src/material/datepicker/datepicker-content.html

```diff
@@ -1,5 +1,8 @@
 <div
   cdkTrapFocus
+  role="dialog"
+  [attr.aria-modal]="true"
+  [attr.aria-labelledby]="_dialogLabelId ?? undefined"
```

Comment:
I _think_ you want `null` instead of undefined here, which will cause Angular to omit the attribute entirely (needs testing to confirm, though)

Reply:
Both null and undefined will work in this case. 

From the [Attribute Binding docs](https://angular.io/guide/attribute-binding#syntax)

> When the expression resolves to null or undefined, Angular removes the attribute altogether.